docs/dotnet/dotnet-http-metrics.md

```diff
@@ -0,0 +1,383 @@
+
+**[1]:** Corresponding `EventCounter` names are `*-connections-current-total`; Meter name is `System.Net.Http`.
+<!-- endsemconv -->
+
+<!-- semconv metric.dotnet.http.client.open_connections(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `http.connection.state` | string | State of HTTP connection in the HTTP connection pool. | `active`; `idle` | Required |
+| [`network.protocol.version`](../general/attributes.md) | string | Version of the application layer protocol used. See note below. [1] | `1.1`; `2.0` | Recommended |
+| [`server.address`](../general/attributes.md) | string | Logical server hostname, matches server FQDN if available, and IP or socket address if FQDN is not known. | `example.com` | Recommended |
+| [`server.port`](../general/attributes.md) | int | Logical server port number | `80`; `8080`; `443` | Recommended |
+| [`server.socket.address`](../general/attributes.md) | string | Physical server IP address or Unix socket address. If set from the client, should simply use the socket's peer address, and not attempt to find any actual server IP (i.e., if set from client, this may represent some proxy server instead of the logical server). | `10.5.3.2` | Recommended: If different than `server.address`. |
+| [`url.scheme`](../url/url.md) | string | The [URI scheme](https://www.rfc-editor.org/rfc/rfc3986#section-3.1) component identifying the used protocol. | `https`; `ftp`; `telnet` | Recommended |
+
+**[1]:** `network.protocol.version` refers to the version of the protocol used and might be different from the protocol client's version. If the HTTP client used has a version of `0.27.2`, but sends HTTP version `1.1`, this attribute should be set to `1.1`.
+
+`http.connection.state` MUST be one of the following:
+
+| Value  | Description |
+|---|---|
+| `active` | active state. |
+| `idle` | idle state. |
+<!-- endsemconv -->
+
+### Metric: `http.client.connection.duration`
+
+<!-- semconv metric.dotnet.http.client.connection.duration(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `http.client.connection.duration` | Histogram | `s` | The duration of outbound HTTP connections. [1] |
+
+**[1]:** Meter name is `System.Net.Http`.
+<!-- endsemconv -->
+
+<!-- semconv metric.dotnet.http.client.connection.duration(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `http.error.reason` | string | "Low-cardinality error reason: `timeout`, `cancellation` or one of [HTTP Request errors](https://github.com/dotnet/runtime/blob/c430570a01c103bc7f117be573f37d8ce8a129b8/src/libraries/System.Net.Http/src/System/Net/Http/HttpRequestError.cs)" [1] | `timeout`; `name_resolution_error`; `secure_connection_error` | Conditionally Required: If request has failed. |
```

Comment:
```suggestion
```

Currently, the connection duration span starts after the successful establishment of the connection & handshake, and we don't count errors. This this is in line with the semantics of the `open_connections` counter and isn't trivial to change, so I recommend to keep this behavior.

Reply:
removed the attribute and added clarification to the description.